lib/gtfs/data.ex

```diff
@@ -274,43 +280,52 @@ defmodule Gtfs.Data do
     )
   end
 
-  @spec timepoint_ids_for_routes([RoutePattern.t()], MapSet.t(Route.id()), trips_by_id()) ::
-          timepoint_ids_by_route()
-  defp timepoint_ids_for_routes(route_patterns, route_ids, trips) do
+  @spec all_timepoints_by_id(binary()) :: timepoints_by_id()
```

Comment:
Could/should this function (and/or the `timepoints_by_id` type) go in `timepoint.ex` instead?

Reply:
I don't have strong feelings about this. This is mirroring many other equivalent functions in this module. I thought in the past you had expressed the opinion that these should be here instead of in e.g. `Timepoint`, but my memory is vague so I could easily be getting that wrong.